### PR TITLE
Fix axis order

### LINKED ISSUE #86: Inconsistency in (row, col) and (col, row) conventions
There is inconsistency on conventions used for arguments and return data order of transforms.
The conventions refer to (row, col) and (col, row) orders. For example, when a transform takes argument `flip_x, flip_y` in this order, it is following (col, row) convention.

The issue is that there are functions that follow those conventions, whereas the image shapes follow (row, col) convention.

Currently, the following codes are related to this issue.

+ flip related (`x_flip`, `y_flip`)
+ expand related (`x_offset`, `y_offset`)
+ crop related (`x_slice`, `y_slice`)
+ shapes used as argument for transforms such as `resize`. (`shape=(H, W)`)

### PR DESCRIPTION
**Pull request by Hakuyume**
Fix #86

### CODE REVIEW

**Comment by yuyu2172** [comment]
Thanks. I think following functions also need to be changed.
+ crop related functions
+ `bbox_resize` and `keypoint_resize`
+ `pad`

**Comment by Hakuyume** [comment]
Thank you. I'll fix them.

**Comment by Hakuyume** [comment]
I fixed some functions.

File: chainercv/transforms/image/random_crop.py [hunk 1]
```diff
@@ -18,28 +18,20 @@ def random_crop(img, output_shape, return_slices=False, copy=False):
         copy (bool): If :obj:`False`, a view of :obj:`img` is returned.
```
Comment (yuyu2172) [inline chainercv/transforms/image/random_crop.py]: Could you fix doc?

**Comment by yuyu2172** [comment]
LGTM except a comment.

File: chainercv/transforms/image/center_crop.py [hunk 2]
```diff
@@ -8,18 +8,18 @@ def center_crop(img, output_shape, return_slices=False, copy=False):
         img (~numpy.ndarray): An image array to be cropped. This is in
             CHW format.
         output_shape (tuple): the size of output image after cropping.
-            This value is :math:`(heihgt, width)`.
+            This value is :math:`(width, heihgt)`.
```
Comment (yuyu2172) [inline chainercv/transforms/image/center_crop.py]: typo

File: chainercv/transforms/image/ten_crop.py [hunk 3]
```diff
@@ -25,16 +25,16 @@ def ten_crop(img, output_shape):
         img (~numpy.ndarray): An image array to be cropped. This is in
             CHW format.
         output_shape (tuple): the size of output images after cropping.
-            This value is :math:`(heihgt, width)`.
+            This value is :math:`(width, heihgt)`.
```
Comment (yuyu2172) [inline chainercv/transforms/image/ten_crop.py]: typo

**Comment by yuyu2172** [comment]
Thanks. LGTM

**Pull request merged**